### DISCUSSION TITLE
Rönd with a service mesh

### DISCUSSION THREAD

**Discussion by gamer22026** [rond-authz/rond#203] [Q&A ~ The Runestone]
Just wondering how Rönd works when there is an additional service mesh sidecar in play (envoy, linkerd, etc). Anyone running it with a service mesh?

**Comment by fredmaggiowski** [comment]
Hi @gamer22026, thanks for reaching out!

For the time being we had no use case with a service mesh integration but we had some ideas. As of now you can add it as an additional sidecar, to act between your mesh sidecar and the application container.

However dealing with three containers per pod may be unconfortable and a potential waste of resources. For this reason we are currently working on making rönd available as a SDK. Having a rönd available as SDK will allow us to:

 - build specific integrations with mesh providers; or:
 - allow you to embed rönd in your codebase (e.g. in a web api middleware)

Does this sound like a good solution for you? Could it work in your use case or do you think we may miss something? If you have any ideas or feedback they are more than welcome!

Thanks

**Reply by fredmaggiowski** [reply]
Hey @gamer22026 ICYMI with [Rönd v1.9.0](https://github.com/rond-authz/rond/releases/tag/v1.9.0) we finally developed an alpha version of the SDK; this means we could work towards service mesh integrations without managing more sidecars :)
